lib/jsdom/living/xmlhttprequest.js

```diff
@@ -590,6 +594,30 @@ module.exports = function createXMLHttpRequest(window) {
     DONE: 4
   });
 
+  function parseContentType(contentType) {
+    const result = { type: "", params: { } };
+
+    if (contentType) {
+      const parts = contentType.split(";");
+      result.type = parts[0];
+      parts.slice(1).forEach(part => {
```

Comment:
Just use a `for (let i = 1; ...)` loop; no need to create a new array just to use `.forEach` on it.